README.md

```diff
@@ -95,6 +95,8 @@ Since currently we are didn't have any default presets, we can't lint it.
 This library written in Haskell with The Haskell Tool Stack.
 Below is an installation and demo for the UNIX-like operating system.
 
+Make sure you have [Haskell Platform](https://www.haskell.org/platform/) installed on your system before running these commands.
```

Comment:
Haskell Platform is not recommended by various Haskell programmer. We use The Haskell Tool Stack instead. You may add them all and I will merge after.